util/helpfulIO.py

```diff
@@ -73,9 +74,12 @@ def _setSimMotor(self, amount: float) -> None:
             self.motor.set(clampedAmount)
             self.simEncoder += (
                 clampedAmount
-                * DCMotor.falcon500().freeSpeed
-                * constants.kTalonEncoderPulsesPerRevolution
-            )  # amount of free speed, free speed is in RPM, convert from revolutions to encodes ticks expected
+                # * 2 * tau # radians per second
+                * DCMotor.falcon500().freeSpeed  # radians per second
+                * constants.kTalonEncoderPulsesPerRadian  # encoder ticks per radian
+                * 1
+                / 50  # 50 updates per second
```

Comment:
Replace with constant from constants.py

Reply:
I will check with Luke on this stuff, it shouldn't be different so I'm not sure what's going on here

Reply:
Can be removed

Reply:
How can I remove the changes in that file alone?